app/sections/posts/posts-list.component.ts

```diff
@@ -0,0 +1,33 @@
+import {Component} from 'angular2/core';
+import {DatePipe} from 'angular2/common';
+import {ContentfulService} from 'ng2-contentful/dist/src/services/contentful.service';
+
+@Component({
+  selector: 'posts',
+  template: <string> require('./posts-list.html'),
+  styles: [<string> require('./posts-list.css')],
+  providers: [ContentfulService],
```

Comment:
I've added it globally to entire app here => https://github.com/Gapminder/gapminder-org/blob/develop/app/index.ts#L24 ( and FYI in the next version it'll be replaced with the internal app service :) ).